bedrock/pocket/templates/pocket/jobs.html

```diff
@@ -12,57 +12,57 @@
 {{ css_bundle('pocket-jobs') }}
 {% endblock %}
 
-{% block page_title %}Jobs{% endblock page_title %}
+{% block page_title %}{{ ftl('pocket-jobs-title') }}{% endblock page_title %}
 
 {% block content %}
   {% include 'pocket/includes/nav.html' %}
   <div class="mzp-l-content mzp-t-content-xl pocket-jobs-wrapper">
-    <h1>Join the Pocket Team</h1>
+    <h1>{{ ftl('pocket-jobs-join-the-pocket') }}</h1>
     <figure>
       {{ high_res_img('img/pocket/jobs/jobs-team.jpg', {'alt': '', 'width': '745', 'height': '408'}) }}
-      <figcaption>Photo by Gaurang Katre</figcaption>
+      <figcaption>{{ ftl('pocket-jobs-photo-by', pocket_photo_author='Gaurang Katre') }} </figcaption>
     </figure>
-    <p>We invite you to become a vital part of our small team in San Francisco. We’re building a company that enables people to save and consume content that is worthy of both their time and attention. By doing so, we want to make it easier to spend time with more high-quality content, no matter how noisy it gets.</p>
-    <p>If you are passionate about using technology to improve people’s lives, we believe Pocket has challenges in front of us that will excite you. We’re hoping you’ll be ready to jump in and get your hands dirty, too. We’re a small team and that means lots of opportunity to own things from start to finish that have tremendous impact on large numbers of people, across many different devices and platforms.</p>
-    <p>Nearly 30 million people have saved more than 3 billion articles and videos to Pocket. Every day, people all over the world are using Pocket to save and read what’s interesting to them, surfacing a unique, human view of what’s actually worth reading and watching on the web.</p>
-    <p>In a world where we are buried by content, where it’s become increasingly harder to find signal in the noise, truth from fiction, and even the value of our press and written word is being challenged, we need a platform where high-quality, thoughtful content and free speech can rise above the rest.</p>
-    <p>If that sounds compelling, we’d love to meet you.</p>
+    <p>{{ ftl('pocket-jobs-we-invite') }}</p>
+    <p>{{ ftl('pocket-jobs-if-you-are-passionate') }}</p>
+    <p>{{ ftl('pocket-jobs-nearly-million') }}</p>
+    <p>{{ ftl('pocket-jobs-in-a-world') }}</p>
+    <p>{{ ftl('pocket-jobs-if-that-sounds') }}</p>
     <section>
-      <h2>Current Positions</h2>
-      <a href="https://www.mozilla.org/en-US/careers/listings/?team=Core%20Product-Pocket">See our latest job openings</a>
+      <h2>{{ ftl('pocket-jobs-current-positions') }}</h2>
+      <a href="https://www.mozilla.org/en-US/careers/listings/?team=Core%20Product-Pocket">{{ ftl('pocket-jobs-see-our-latest') }}</a>
```

Comment:
Nit: Since this is considered an external link (out of the Pocket website), I think it should be worth adding target="_blank" rel="external noopener"?

Reply:
Good point!